fjord/base/forms.py

```diff
@@ -1,8 +1,26 @@
 from django.forms import fields
+from django.forms import widgets
 
 from fjord.base.validators import EnhancedURLValidator
 
 
+class StringListField(fields.CharField):
+    widget = widgets.Textarea()
+
+    def prepare_value(self, value):
```

Comment:
Could you add docstrings to these methods to note which of these methods deal with from-the-database data and which ones deal with from-python-objects data?